Docs/development/jasp-human-guide.md

```diff
@@ -29,6 +29,34 @@ Hence, in developing an analysis, it is good to think through each and every sor
 
 Error messages can either be placed over the top of the results table, or put in the footnotes. In general, an error which makes the whole analysis meaningless, should be placed over the top of the results table. A t-test where the user has specified an independent variable with three levels would be one such example. In contrast, if the error only affects one or a handful of values in the table, then an NaN should be placed in that cell, and a footnote marker added.
 
+### Writing for internationalization
+
+Internationalization(i18n) is the adapting of module to different languages. An inspiring news is that JASP is moving towards i18n, which means that the analysis and modules you developed will be translated into different languages. Therefore, it is necessary to pay attention to the readability and flexibility of the message during writing.
+Passing your message with [gettext()](https://en.wikipedia.org/wiki/Gettext), JASP will generate the string  to be translated automatically in a `.po` file.
+
+**Placeholders**
+
+If you have multiple placeholders, you can provide them inside `gettext()` as `$1`, `$2`, and `$3` and so on. 
```

Comment:
You may have multiple placeholders for one message: 
`gettextf("Number of factor levels is %s in %s", amount, variables)`.
The problem here is that the order of the arguments depends on the language, and that for some languages the `variables` argument will be used before the `amount` argument.
Therefore it is important to specify the argument order:
`gettextf("Number of factor levels is %1$s in %2$s", amount, variables)`
So that the translation in some language can place the `variables` argument first:
`Dans %2$s, le nombre de niveaux des facteurs est %1$s`